src/components/features/NewFurniture/NewFurnitureContainer.js

```diff
@@ -4,10 +4,15 @@ import NewFurniture from './NewFurniture';
 
 import { getAll } from '../../../redux/categoriesRedux.js';
 import { getNew } from '../../../redux/productsRedux.js';
+import { setFavoriteProduct } from '../../../redux/favoritesRedux';
 
 const mapStateToProps = state => ({
   categories: getAll(state),
   products: getNew(state),
 });
 
-export default connect(mapStateToProps)(NewFurniture);
+const mapDispatchToProps = dispatch => ({
+  setFavoriteProduct: product => dispatch(setFavoriteProduct(product)),
+});
```

Comment:
Are you sure that `product` is a good name for this param? You don't really provide the whole product here. It's only id, isn't it? I think `productId` would be better here :)